priv/perf/apps/load_test/lib/ethereum/account.ex

```diff
@@ -57,7 +57,7 @@ defmodule LoadTest.Ethereum.Account do
   end
 
   defp compute_public_key(private_key) do
-    {:ok, _} = :libsecp256k1.ec_pubkey_create(private_key, :uncompressed)
+    ExSecp256k1.create_public_key(private_key)
```

Comment:
can we match with {:ok, ...} ? so the function would fail in case of an error

Reply:
the caller does that already: https://github.com/omgnetwork/elixir-omg/pull/1789/files#diff-73d593649e7db3ed9a7e11a378fc2c56aee72864c99df4dd18a407e71460ac7dR39